src/static/css/index.css

```
@@ -682,3 +628,22 @@ p {
     margin: 0;
   }
 }
+
+@media (max-width: 900px) {
+  .intro-year {
+    display: block;
+    color: #f7f779;
+    font-size: min(190px, 40vw);
```

Was wondering why you hadn't answered my nit as well - but just realised I hadn't actually sent it!

Should we add a fallback font size here since `min()` [isn't quite universal yet](https://caniuse.com/#feat=css-math-functions)?

```suggestion
    font-size: 190px;
    font-size: min(190px, 40vw);
```

---

Sure, good find!